src/graphql/error-map.ts

```diff
@@ -94,6 +94,10 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
       message = "An amount is required to complete payment"
       return new ValidationInternalError({ message, logger: baseLogger })
 
+    case "InvalidWithdrawFeeError":
```

Comment:
This should never be exposed to the enduser. If we get here the app is mis-configured. Returning the `ValidationInternalError` is notifying the caller that they have made a mistake. This is a bad state and should probably be part of the `unhandled` errors further down.

Reply:
Fixed